dropbox/oauth.py

```diff
@@ -120,11 +130,18 @@ def __repr__(self):
 
 class DropboxOAuth2FlowBase(object):
 
-    def __init__(self, consumer_key, consumer_secret, locale=None, token_access_type='legacy',
-                 scope=None, include_granted_scopes=None):
+    def __init__(self, consumer_key, consumer_secret=None, locale=None, token_access_type='legacy',
+                 scope=None, include_granted_scopes=None, pkce_method=None):
```

Comment:
I think it make no sense to ever provide plain here. We should always use s256. 

And still leave a boolean like "use_pkce"